data.sql

```diff
@@ -1,14 +1,73 @@
 WITH sample data. */
-INSERT INTO animals VALUES (1, 'Agumon', '02/03/2020', 0, 'T', 10.23);
-INSERT INTO animals VALUES (2, 'Gabumon', '09/15/2018', 2, 'T', 8);
-INSERT INTO animals VALUES (3, 'Pikachu', '01/07/2021', 1, 'F', 15.04);
-INSERT INTO animals VALUES (4, 'Devimon', '05/12/2017', 5, 'T', 11);
+-- ANIMALS
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Agumon', '02/03/2020', 0, 'T', 10.23);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Gabumon', '09/15/2018', 2, 'T', 8);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Pikachu', '01/07/2021', 1, 'F', 15.04);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Devimon', '05/12/2017', 5, 'T', 11);
 ALTER TABLE animals add column species char(30);
-INSERT INTO animals VALUES (5, 'Charmandar', '02/08/2020', 0, 'F', -11);
-INSERT INTO animals VALUES (6, 'Plantom', '11/15/2021', 2, 'T', -5.7);
-INSERT INTO animals VALUES (7, 'Squirtle', '04/02/1993', 3, 'F', -12.13);
-INSERT INTO animals VALUES (8, 'Angemon', '06/12/2005', 1, 'T', -45);
-INSERT INTO animals VALUES (9, 'Boarmon', '06/7/2020', 7, 'T', 20.4);
-INSERT INTO animals VALUES (10, 'Blossom', '10/13/1998', 3, 'T', 17);
-INSERT INTO animals VALUES (11, 'Ditto', '05/14/2022', 4, 'F', 22);
\ No newline at end of file
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Charmandar', '02/08/2020', 0, 'F', -11);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Plantom', '11/15/2021', 2, 'T', -5.7);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Squirtle', '04/02/1993', 3, 'F', -12.13);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Angemon', '06/12/2005', 1, 'T', -45);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Boarmon', '06/7/2020', 7, 'T', 20.4);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Blossom', '10/13/1998', 3, 'T', 17);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Ditto', '05/14/2022', 4, 'F', 22);
+
+-- OWNERS
+insert into owners(full_name, age) values('Sam Smith', 34);
+insert into owners(full_name, age) values('Jennifer Orwell', 19);
+insert into owners(full_name, age) values('Bob', 45);
+insert into owners(full_name, age) values('Melody Pond', 77);
+insert into owners(full_name, age) values('Dean Winchester', 14);
+insert into owners(full_name, age) values('Jodie Whittaker', 38);
+
+-- SPECIES
+insert into species(name) values('Pokemon');
+insert into species(name) values('Digimon');
+
+-- Queries
+begin;
+alter table animals
+drop column species;
+commit;
+
+begin;
+alter table animals
+add column species_id int references species(id);
+
+alter table animals
+add column owner_id int references owners(id);
+commit;
```

Comment:
- [ ] Dear @Shakir-Hussain12, Please remember everything that creates or `changes` the structure of the table should be saved in the `schema.sql` file. as a result of this these `ALTER` statements should be written inside your `schema.sql` file.

Reply:
Sorry @TadesseDev , It took a while mainly because of some electricity issues.

Reply:
That's all right @Shakir-Hussain12 👏